packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -137,6 +176,29 @@ export class CfnInclude extends core.CfnElement {
     return ret;
   }
 
+  /**
+   * Returns the NestedStack with name logicalId.
+   * For a nested stack to be returned by this method, it must be specified in the CfnIncludeProps.
+   * @param logicalId the ID of the stack to retrieve, as it appears in its template.
+   */
+  public getNestedStack(logicalId: string): IncludedNestedStack {
+    if (!this.nestedStacks[logicalId] || !this.nestedStackTemplates[logicalId]) {
```

Comment:
I find this condition a little confusing.

Is it possible for something to be in `this.nestedStacks`, but somehow _not_ be in `nestedStackTemplates`?